app-text/poppler/poppler23-23.09.0.recipe

```diff
@@ -296,14 +296,14 @@ REQUIRES_devel="
 #	devel:libnss3$secondaryArchSuffix
 	devel:libopenjp2$secondaryArchSuffix
 	devel:libpng16$secondaryArchSuffix
-	devel:libQt5Core$secondaryArchSuffix
-	devel:libQt5Gui$secondaryArchSuffix
-	devel:libQt5Widgets$secondaryArchSuffix
-	devel:libQt5Xml$secondaryArchSuffix
-	devel:libQt6Core$secondaryArchSuffix
-	devel:libQt6Gui$secondaryArchSuffix
-	devel:libQt6Widgets$secondaryArchSuffix
-	devel:libQt6Xml$secondaryArchSuffix
+#	devel:libQt5Core$secondaryArchSuffix
+#	devel:libQt5Gui$secondaryArchSuffix
+#	devel:libQt5Widgets$secondaryArchSuffix
+#	devel:libQt5Xml$secondaryArchSuffix
+#	devel:libQt6Core$secondaryArchSuffix
+#	devel:libQt6Gui$secondaryArchSuffix
+#	devel:libQt6Widgets$secondaryArchSuffix
+#	devel:libQt6Xml$secondaryArchSuffix
 	devel:libtiff$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
```

Comment:
Only poppler-glib mentions devel packages when checking with `pkg-config --cflags`, not even regular poppler or poppler-cpp mention any extras.